src/Dotnet.Script.Core/Dotnet.Script.Core.csproj

```diff
@@ -18,12 +18,35 @@
     <GenerateAssemblyConfigurationAttribute>false</GenerateAssemblyConfigurationAttribute>
     <GenerateAssemblyCompanyAttribute>false</GenerateAssemblyCompanyAttribute>
     <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
+    <Version>0.11.0</Version>
```

Comment:
The version properties are confusing.

Related:
- https://github.com/Microsoft/msbuild/issues/1952
- https://twitter.com/adamralph/status/850238043709521924